app/controllers/crawls_controller.rb

```diff
@@ -61,6 +61,10 @@ def destroy
     end
   end
 
+  def modalCheck
```

Comment:
Ruby/Rails methods are _ by convention, so please do modal_check instead of modalCheck.  Also, lets name this more about what it is doing and less about how it is implemented.  age_verified??